invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/RDMDepositForm/RDMDepositForm.js

```diff
@@ -40,6 +30,7 @@ const defaultRecord = {
     metadata_restricted: false,
     files_restricted: false,
     owners: [1],
+    access_right: "open",
```

Comment:
I removed that one because it didn't show up as selected (but was there) and so when the payload was sent it wasn't clear what value was being sent. Not providing it also allows for the error to show up.

Reply:
Fixed that here https://github.com/inveniosoftware/react-invenio-deposit/commit/8cb590fb23185f537517c634f1866c8e91b93309 . It's a temporary solution as the `GroupField` component should pass down formik props :)